rwslib/tests/test_audit_event.py

```diff
@@ -0,0 +1,171 @@
+from rwslib.extras.audit_event import parser
+import unittest
+import os
+
+class MockEventer:
+
```

Comment:
A small nit but a class doc here would be useful to explain what its for - just that this and the below are good examples of audit event sinks - not sure they are even really "mocks" since they implement the event interface expected but that isn't a criticism.